docs/_includes/Running-Flink-on-Alluxio/alluxio-client-properties.md

```diff
@@ -0,0 +1,3 @@
+```yaml
+env.java.opts: -Dalluxio.user.file.writetype.default=CACHE_THROUGH
```

Comment:
we're trying to move away from creating external code snippets - it makes reading and editing the docs source code much harder. Please inline this snippet

Reply:
Done.